core_backend/tests/api/test_manage_content.py

```diff
@@ -163,113 +163,6 @@ async def test_delete_content(
         assert response.status_code == 200
 
 
-class TestAuthManageContent:
```

Comment:
I worry that we are removing readonly from tests but not fully from functionality. What are you thoughts?